crates/aptos/src/common/types.rs

```diff
@@ -41,6 +46,8 @@ use std::{
 };
 use thiserror::Error;
 
+const MAX_POSSIBLE_GAS_UNITS: u64 = 1_000_000;
```

Comment:
@vgao1996 is there an easier way for me to pull this from onchain?

Reply:
You need to fetch the resource representing the on-chain gas schedule. Do we have this kind of API available arleady?